Refer to: https://dwm.suckless.org/patches/focusmonmouse/
Original: https://dwm.suckless.org/patches/focusmonmouse/dwm-focusmonmouse-6.2.diff
Authors:
    - Filip Seman - seman.filip@gmail.com

--- a/dwm.c
+++ b/dwm.c
@@ -877,6 +877,7 @@ focusmon(const Arg *arg)
 	if ((m = dirtomon(arg->i)) == selmon)
 		return;
 	unfocus(selmon->sel, 0);
+	XWarpPointer(dpy, None, m->barwin, 0, 0, 0, 0, m->mw / 2, m->mh / 2);
 	selmon = m;
 	focus(NULL);
 	if (selmon->sel)
